pkgdb2/lib/__init__.py

```diff
@@ -291,7 +335,8 @@ def set_acl_package(session, pkg_name, pkg_branch, pkg_user, acl, status,
         raise PkgdbException('No collection found by the name of %s'
                              % pkg_branch)
 
-    if not pkgdb2.is_pkg_admin(session, user, package.name, pkg_branch):
+    if not force and not pkgdb2.is_pkg_admin(
```

Comment:
On Wed, Jan 21, 2015 at 07:52:20AM -0800, Ralph Bean wrote:
>    In pkgdb2/lib/__init__.py:
> 
>  > @@ -291,7 +335,8 @@ def set_acl_package(session, pkg_name, pkg_branch, pkg_user, acl, status,
>  >          raise PkgdbException('No collection found by the name of %s'
>  >                               % pkg_branch)
>  > 
>  > -    if not pkgdb2.is_pkg_admin(session, user, package.name, pkg_branch):
>  > +    if not force and not pkgdb2.is_pkg_admin(
> 
>    force seems like a bad idea.
> 
>    So, anyone can create ACLs without being an admin?

I agree that forces is a bad idea, but since new branch request for Fedora
branches are automatically approved and all the person needs to make such
request is being in the packager group, I needed a way to automatically grant
that person its ACLs without intervention from an admin.

So the only place where force is used is:
https://github.com/fedora-infra/pkgdb2/blob/admin_actions/pkgdb2/lib/__init__.py#L1739